raiden/network/pathfinding.py

```diff
@@ -57,12 +83,31 @@ def get_random_service(
     return url, address
 
 
+class PFSConfiguration(typing.NamedTuple):
```

Comment:
At some point, we're moving to dataclasses... so maybe it's a good idea to just start using them instead of introducing more namedtuples

Reply:
I changed this back to a `NamedTuple` now. @LefterisJP is right in that we should keep using `NamedTuple` for immutable types.